5.4.2 release notes update (#154)
neuvector/docs: versioned_docs/version-5.4/14.releasenotes/01.5x/01.5x.md

- Provide an internal certificate using the existing methods:
  https://open-docs.neuvector.com/deploying/production/internal

##### Upgrading from Previous Versions to 5.4.2:

Please create and configure internal certificates from the scanner for the controller, enforcer, and registry-adapter to achieve a rolling update without losing data. It is still recommended to take a backup of your configuration before upgrading. 

> The following steps are needed only for the deployments that used yaml file. Upgrading using helm doest not need these steps because internal certificates are created by these flags by default `internal.autoGenerateCert` and `internal.autoRotateCert` by helm. 

Reviewer: ```suggestion
The following steps are only needed if your deployment uses a `.yaml` file. Upgrading using Helm does not need these additional steps due to the internal certificates getting created by default via the following flags: `internal.autoGenerateCert` and `internal.autoRotateCert`. 
```

Sample 5.4.2 yaml with internal certificate configured:
*https://github.com/neuvector/manifests/blob/main/kubernetes/5.4.0/neuvector-k8s.yaml* 

> Additional info about internal certificates

Reviewer: ```suggestion
Additional information about internal certificates:
```
